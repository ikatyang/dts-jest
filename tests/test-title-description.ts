declare const test_title_description_1: () => {result: string};
declare const test_title_description_2: () => {result: number};
declare const test_title_description_3: () => {result: symbol};

// @dts-jest
test_title_description_1(

);

// @dts-jest
test_title_description_2();

// @dts-jest optional-description
test_title_description_3();
