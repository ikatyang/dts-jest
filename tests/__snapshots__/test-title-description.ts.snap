// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`optional-description 1`] = `"{ result: symbol; }"`;

exports[`test_title_description_1(
    
    ) 1`] = `"{ result: string; }"`;

exports[`test_title_description_2() 1`] = `"{ result: number; }"`;
